catroid/src/main/res/values-sr-rCS/strings.xml

```diff
@@ -1809,7 +1809,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">Operatori za poređenje</string>
     <string name="formula_editor_object_general">Opšta svojstva</string>
     <string name="formula_editor_object_movement">Svojstva pokreta</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations